.github/workflows/test-script.yml

```diff
@@ -11,28 +11,33 @@ name: Test Script
 jobs:
   test:
     strategy:
-      fail-fast: true
+      fail-fast: false
       matrix:
         wordpress: ["latest", "trunk", "6.0"]
+        install-core-tests: [0, 1]
     runs-on: ubuntu-latest
-    name: "Test Script ${{ matrix.wordpress }}"
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ROOT_PASSWORD: "root"
+        ports:
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+    name: "Test Script ${{ matrix.wordpress }} - Core Suites: ${{ matrix.install-core-tests }}"
     env:
-      CACHEDIR: /tmp/test-cache
-      WP_CORE_DIR: /tmp/wordpress
+      WP_CORE_DIR: '/tmp/wordpress'
       WP_VERSION: ${{ matrix.wordpress }}
-      WP_DB_HOST: 127.0.0.1
-      WP_DB_USER: root
-      WP_DB_PASSWORD: 'password'
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - name: "Run install-wp-tests.sh"
+      - name: "Run install-wp-tests.sh and check installation"
         shell: bash
         run: |
-          rm -rf $WP_CORE_DIR
-          bash install-wp-tests.sh wordpress_unit_tests "$WP_DB_USER" "$WP_DB_PASSWORD" "$WP_DB_HOST" $WP_VERSION true
-      - name: "Check Installation"
-        shell: bash
-        run: bash bin/test-script.sh
+          export INSTALL_WP_TEST_DEBUG=1
+          export WP_INSTALL_CORE_TEST_SUITE=${{ matrix.install-core-tests }}
+
+          bash install-wp-tests.sh wordpress_unit_tests root root localhost "$WP_VERSION" false true true
```

Comment:
As a human, I wouldn't mind a comment above this line describing the arguments, even if it's just
```
install-wp-tests.sh <db-name> <db-user> <db-pass> <db-host> <wp-version> <skip-database-creation> <install-mu-plugins> <install-memcached>
```